### PR TITLE
Create cards.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
card-vertical

Created following variations of card :
-->vertical card with text, image and badge ,icon, close button.
-->Horizontal card with txt and image
-->Card with text overlay.
-->Only Text card

### CODE REVIEW

File: Components/cards.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+.card-horizontal {
+  max-width: 800px;
+  padding: 0.5rem 1.8rem 0 1.5rem;
+  display: flex;
+  box-shadow: 1px 4px 5px 3px var(--light-secondary-color);
+  position: relative;
+}
+.card-horizontal .card-img {
+  width: 120px;
```
Comment (chanchal16) [inline Components/cards.css]: Don't use absolute units if you plan to make them responsive.

File: Components/cards.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  border: 1px solid var(--complementary-text-color);
+}
+.card-favourite-icon {
+  background-color: #fff;
+  padding: 0.4rem;
+  border-radius: 50%;
+  position: absolute;
+  right: 1rem;
+  top: 0.5rem;
+  font-size: 20px;
```
Comment (aryan-13) [inline Components/cards.css:32]: Rems are preffered

File: Components/cards.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  max-width: 350px;
+}
+.text-card .card-action {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 50px;
```
Comment (aryan-13) [inline Components/cards.css]: Here also

File: Components/cards.css (hunk 1 above)
Comment (aryan-13) [inline Components/cards.css]: You can replace absolute units with relative units

File: Components/cards.css [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+.card-horizontal {
+  max-width: 800px;
+  padding: 0.5rem 1.8rem 0 1.5rem;
+  display: flex;
+  box-shadow: 1px 4px 5px 3px var(--light-secondary-color);
+  position: relative;
+}
+.card-horizontal .card-img {
+  width: 120px;
+  height: 150px;
```
Comment (aryan-13) [inline Components/cards.css]: Here too

**Review by aryan-13 — COMMENTED** [review]
Relative units are better for responsiveness and are preferred over the absolute ones

File: Components/cards.css (hunk 3 above)
Comment (Radheshyam1234) [reply Components/cards.css]: Changes made

File: Components/cards.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/cards.css]: Changes made

**Pull request merged**